test/helpers/story/drivers/dapp-story-test-driver.ts

```diff
@@ -113,7 +118,20 @@ export default class DappStoryTestDriver extends StoryTestDriver {
       }
       case 'POOL_TOGETHER': {
         const newTest = new Test('POOL_TOGETHER Lend DAPP', (async () => {
-          expect(1).to.equal(1)
+          const borrower = await getNamedSigner('borrower')
+          const loan = await LoanStoryTestDriver.getLoan(hre, borrower)
+          const { details, diamond } = loan
+          let shouldPass = true
+          const borrowedAmount = (await details.lendingToken.balanceOf(await diamond.getLoanEscrow(details.loan.id))).toString()
+          if(borrowedAmount != details.loan.borrowedAmount.toString()) shouldPass = false
```

Comment:
what specific about this check makes the `shouldPass = false`?

Reply:
what we talked about, with the borrowed amount not matching what it's depositing. as a part of that loan has already been used